### PR TITLE
Add Dogecoin.link with Dogedir.com on the Store tab

### PR DESCRIPTION
**Pull request by jflaloux**
Hi all,
New to github, I hope that I'm doing everything correctly.
Dogecoin.link is a new User-generated Dogecoin Directory.
I think that it's good for new users to have more alternatives on the main site.
Any doubts, I'm at disposition!
Thx!

### CODE REVIEW

**Comment by DomT4** [comment]
Your formatting is indeed correct for Github. You might want to tweak line 285 to reflect having multiple links rather than just the one though.

**Comment by jflaloux** [comment]
Great idea! Done!

**Comment by jflaloux** [comment]
Who will decide to go forward with this request?  
Thx!

**Comment by DomT4** [comment]
Either @ummjackson or @PowerLemons. They have the ability to merge PRs into the master.

**Comment by jflaloux** [comment]
Shall I tweet them?

**Comment by DomT4** [comment]
They'll have received a Github notification from me using the @ tag here a moment ago, so they'll see this thread the next time either one logs in here.

**Comment by jflaloux** [comment]
Thanks a lot for the help @DomT4  !

**Comment by PowerLemons** [comment]
Hello @jflaloux,
I do not know if Dogecoin.com would benefit with another Dogecoin directory website. What does Dogecoin.link offer that Dogedir doesn't? The user doesn't need to look at two sites which serves the same purpose.

I would really like to hear from you about this, because I and the Dogecoin community appreciates what you're doing and it would be great to add your site in =).

**Comment by jflaloux** [comment]
Hey @PowerLemons 
Thank you for the response. 

Here is my opinion:

Clicking the "Stores" icon on Dogecoin.com is kind of confusing.  
There is 3 stores listed : 
-ShopDoge : that seems down
-BitcoinShop : That's actually call "Bitcoin"shop
-Dogeoutlet : Great shop.. I love it

There is more than Markets, there is auction sites, services, faucets..
I totally understand that's it's difficult for you to take care of this list of shops, every time there is a new site or a site down.
What I want to say is that, taking care of listings of site is a big job (I know about it).
It's definitely not a static list and it's a job on it's own to take care of it. That's why there is directories.

Dogecoin.link is user generated, meaning that when user discover a new site, they can add it and receive Dogecoin to do so.
The great advantage is that it's fast and reactive, by example, DogecoinMarket.io was launched yesterday, and it's already on Dogecoin.link : http://www.dogecoin.link/item/dogecoinmarket-io/

Now, comparing to Dogedir, it's pretty similar that's for sure. Dogedir is more sober and direct. Dogecoin.link is colourful with image of site and reviews. It's two different styles and each will prefer one. But I'd say it's always better to offer different option for the final user. Also Dogedir as much more listings, but that's also because Dogecoin.link was launched 3weeks ago.

In conclusion, I think it would be a plus for the users to have a renewed "Stores" tab.

Thank you again for the consideration.
I'm at your disposition.

**Comment by jflaloux** [comment]
Any Follow up @PowerLemons ?

**Comment by PowerLemons** [comment]
Merged. Thank you very much for contributing!
